pages/linux/cu.md

```diff
@@ -0,0 +1,24 @@
+# cu
+
+> **Call Up** another system and act as a dial-in / serial terminal or perform file transfers with no error checking.
+> More information: <https://manned.org/cu>.
+
+- Open a given serial port:
+
+`sudo cu --line {{/dev/ttyUSB0}}`
+
+- Open a given serial port with a given baud rate:
+
+`sudo cu --line {{/dev/ttyUSB0}} --speed {{115200}}`
+
+- Open a given serial port with a given baud rate and echo characters locally (half-duplex mode):
+
+`sudo cu --line {{/dev/ttyUSB0}} --speed {{115200}} --halfduplex`
+
+- Open a given serial port with a given baud rate, parity, no hardware or software flow control:
+
+`sudo cu --line {{/dev/ttyUSB0}} --speed {{115200}} --parity={{even|odd|none}} --nortscts --nostop`
+
+- When in a connection exit the `cu` session by typing:
```

Comment:
```suggestion
- Exit the `cu` session when in connection:
```
I dont know if we support this kind of examples, but it should be in imperative case anyway.